### PR TITLE
Support for Astra's Rocket and Electron

### PR DESCRIPTION
**Pull request by Vieju17**
This request aims to incorporate RO configs for Beale and EStreetRockets' Ziegler Launch System mod which adds Astra's Rocket 3.0 and 3.3 launch vehicles and Tygoo7's Dodo Labs mod which adds the Electron rocket.

### CODE REVIEW

**Comment by Vieju17** [comment]
They are both completely finished now. The burn times and dimensions should fit real life specifications though many of them are approximations. They'll be changed if new sources or evidences are presented, of course. 

(IDK what the failed check is about)

File: GameData/RealismOverhaul/RO_SuggestedMods/DodoLabs/Electron_RO.cfg [hunk 1]
```diff
@@ -0,0 +1,651 @@
+//Sources:	https://space.skyrocket.de/doc_lau/electron.htm
+			https://www.rocketlabusa.com/space-systems/photon/
+			https://www.rocketlabusa.com/launch/electron/
+			https://www.rocketlabusa.com/updates/the-kick-stage-responsible-orbital-deployment/
+			https://en.wikipedia.org/wiki/Rocket_Lab_Electron
+
```
Comment (Capkirk123) [inline GameData/RealismOverhaul/RO_SuggestedMods/DodoLabs/Electron_RO.cfg:6]: These need to be commented out. Please do not leave plain text in .cfgs

**Review by Capkirk123 — CHANGES_REQUESTED** [review]
You left some lines uncommented, throwing the config error

File: GameData/RealismOverhaul/RO_SuggestedMods/DodoLabs/Electron_RO.cfg (hunk 1 above)
Comment (Vieju17) [reply GameData/RealismOverhaul/RO_SuggestedMods/DodoLabs/Electron_RO.cfg:6]: Oops, I overlooked that, thanks! It is fixed now.

**Review by Capkirk123 — APPROVED** [review]

**Pull request merged**